com.unity.ml-agents/Runtime/Actuators/IBuiltInActuator.cs

```diff
@@ -0,0 +1,30 @@
+namespace Unity.MLAgents.Actuators
+{
+    /// <summary>
+    /// Identifiers for "built in" actuator types.
+    /// These are only used for analytics, and should not be used for any runtime decisions.
+    ///
+    /// NOTE: Do not renumber these, since the values are used for analytics. Renaming is allowed though.
+    /// </summary>
+    public enum BuiltInActuatorType
+    {
+        Unknown = 0,
+        // VectorActuator used by the Agent
+        AgentVectorActuator = 1,
+        VectorActuator = 2,
+        Match3Actuator = 3
+    }
```

Comment:
TODO (depending on whether https://github.com/Unity-Technologies/ml-agents/pull/4881 lands before or after this) - add enums for the actuators defined there.